samples/unit-tests/maps/drilldown/demo.js

```diff
@@ -359,3 +359,84 @@ QUnit.test('Map drilldown animation for GeoJSON maps', async assert => {
         TestUtilities.lolexUninstall(clock);
     }
 });
+
+QUnit.test(
+    '#20886, Map drilldown datalabel mouse tracking',
+    async assert => {
+        const world = await fetch(
+                'https://code.highcharts.com/mapdata/custom/world-continents.topo.json'
+            ).then(response => response.json()),
+            africa = await fetch(
+                'https://code.highcharts.com/mapdata/custom/africa.topo.json'
+            ).then(response => response.json());
+
+        // Create the chart
+        const chart = Highcharts.mapChart('container', {
+                chart: {
+                    animation: false,
+                    events: {
+                        drillupall() {
+                            const chart = this;
+                            assert.ok(
+                                // eslint-disable-next-line no-underscore-dangle
+                                chart.series[0]._hasTracking === false,
+                                'Tracking should be false before chart ' +
+                                're-renders on \"drillupall\" event.'
+                            );
+                        }
+                    }
+                },
+                mapNavigation: {
+                    enabled: true,
+                    buttonOptions: {
+                        verticalAlign: 'bottom'
+                    }
+                },
+                plotOptions: {
+                    map: {
+                        dataLabels: {
+                            enabled: true,
+                            format: '{point.name}'
+                        }
+                    }
+                },
+                series: [{
+                    mapData: world,
+                    custom: {
+                        startPos: void 0
+                    },
+                    data: [{
+                        'hc-key': 'af',
+                        value: 1,
+                        drilldown: 'africa'
+                    }]
+                }],
+                drilldown: {
+                    animation: false,
+                    mapZooming: true,
+                    series: [{
+                        id: 'africa',
+                        mapData: africa
+                    }]
+                }
+            }),
+            done = assert.async();
+
+        chart.series[0].points
+            .find(p => p['hc-key'] === 'af').firePointEvent('click');
+
+        setTimeout(() => {
+            chart.drillUp();
+
+            setTimeout(() => {
+                assert.ok(
+                    chart.series[0].dataLabelsGroup
+                        .hasClass('highcharts-tracker'),
+                    'Data label group should have tracking after drillup.'
+                );
```

Comment:
Do we need an entirely new test (QUnit) for checking only one class on the data labels group? I think we can use the existing test from above and put this `assert` somewhere there.

Reply:
After having fiddled with this for a while I think it's better to have it as a separate test, as the above tests are rather convoluted and have tons of stuff going on at different times (like turning off animations so on).
I have to have the animation on (cause that's when the issue occurs) so that also limits the test's I can modify.

FYI, I test two things here, that tracking has been turned off when it should be (before drillup and before the mapChart has re-rendered) and that tracking has been applied after rendering.

Reply:
Ok! Now, after the last commit I can some some commented tests, is this desired or overlooked?